### DISCUSSION TITLE
Zero or negative cell volume detected

### DISCUSSION THREAD

**Discussion by gocfd** [nextfoam/baram#118] [Q&A]
Hello,

is there any option in Baram mesh to supress cells with negative volume after boundary layers generation, or somehow to tweak the parameters for snapping to avoid negative cells?
I am unable to run the simulation in Baram flow, most probalby because of negative volume cells.

Thanks in advance.

**Comment by kdg5424** [comment]
Hello @gocfd. This is Donggyu from NEXTFOAM.

Sorry for the delay. It was thanksgiving day of Korea until yesterday.
The generation of the boundary layer is the last step of the baramMesh. So there is no way to avoid negative cells after boundary layers.
Could you check the face of the wall or any other boundaries?

Thanks

Best regards

Donggyu kim
